pre_commit_dbt/utils.py

```diff
@@ -439,8 +439,12 @@ def add_related_sqls(
             and node.get("config", {}).get("materialized") == "ephemeral"
         ):
             continue
-        if "patch_path" in node and dbt_patch_path in node["patch_path"]:
-            if ".sql" in node["original_file_path"]:
+        if (
```

Comment:
Maybe `if node.get("patch_path", {}).get("dbt_patch_path")` is simpler?

Reply:
Wouldn't work, `dbt_patch_path` is the variable (yml path) we are looking for, not the key-name (`.get("dbt_patch_path")` will always return nothing

and `node.get("patch_path")` returns a string, not a dictionary, so `.get()` on it will never work

Reply:
yeah sorry, `if node.get("patch_path", {}).get(dbt_patch_path)`

Reply:
`node.get("patch_path")` is (if exists) a string, `.get()` on it's content would break

Reply:
ok, I see, `if dbt_patch_path in node.get("patch_path", ""):`